blip-project/src/components/SignUp, Login/number.jsx

```
@@ -0,0 +1,9 @@
+import Input from "./input";
+
+export const Number = ({ value, onChange, placeholder, ...props }) => {
```

Biome (1.9.4)

[error] Do not shadow the global "Number" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)